WordPress/Classes/ViewRelated/Feature Highlight/Tooltip.swift

```diff
@@ -251,32 +272,72 @@ final class Tooltip: UIView {
         ])
     }
 
+    private func addShadow() {
+        containerView.layer.shadowColor = UIColor.black.cgColor
+        containerView.layer.shadowOffset = CGSize(width: 0, height: 2)
+        containerView.layer.shadowOpacity = 0.5
+    }
+
     private static func height(
         withTitle title: String?,
-        message: String?) -> CGFloat {
-            var totalHeight: CGFloat = 0
+        message: String?
+    ) -> CGFloat {
+        var totalHeight: CGFloat = 0
 
-            totalHeight += Constants.Spacing.contentStackViewTop
+        totalHeight += Constants.Spacing.contentStackViewTop
 
-            if let title = title {
-                totalHeight += title.height(withMaxWidth: Constants.maxWidth, font: Constants.Font.title)
-            }
+        if let title = title {
+            totalHeight += title.height(withMaxWidth: Constants.maxContentWidth, font: Constants.Font.title)
+        }
 
-            totalHeight += Constants.Spacing.contentStackViewInterItemSpacing
+        totalHeight += Constants.Spacing.contentStackViewInterItemSpacing
 
-            if let message = message {
-                totalHeight += message.height(withMaxWidth: Constants.maxWidth, font: Constants.Font.message)
-            }
+        if let message = message {
+            totalHeight += message.height(withMaxWidth: Constants.maxContentWidth, font: Constants.Font.message)
+        }
+
+        totalHeight += Constants.Spacing.buttonStackViewHeight
+        totalHeight += Constants.Spacing.contentStackViewBottom
+
+        return totalHeight
+    }
 
-            totalHeight += Constants.Spacing.buttonStackViewHeight
-            totalHeight += Constants.Spacing.contentStackViewBottom
+    private static func width(
+        title: String?,
+        message: String?,
+        primaryButtonTitle: String?,
+        secondaryButtonTitle: String?
+    ) -> CGFloat {
 
-            return totalHeight
+        let titleWidth: CGFloat
+        if let title = title {
+            titleWidth = title.width(withMaxWidth: Constants.maxContentWidth, font: Constants.Font.title)
+        } else {
+            titleWidth = 0
+        }
```

Comment:
Optional stylistic nitpick, but this could be simplified like so:
```suggestion
        let titleWidth = title?.width(withMaxWidth: Constants.maxContentWidth, font: Constants.Font.title) ?? 0
```